tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "noImplicitAny": true,
     "allowJs": false,
     "rootDir": "src",
+    "outDir": "dist"
```

Comment:
This fixed a major issue. Without this `outDir` setting, the `build` script was running `tsc` but tsc was putting its output `.d.ts` files into `./node_modules/@edx/typescript-config/dist/` !